src/content/en/2020/css.md

```diff
@@ -115,22 +126,40 @@ Only a few of the top classes were presentational, with most of those being eith
 
 Despite IDs being discouraged these days in some circles due to their much higher specificity, most websites still use them, albeit sparingly. Fewer than half of pages used more than one ID in any of their selectors (had a max specificity of (1,x,y) or less) and nearly all had a median specificity that did not include IDs (0,x,y).
 
+{# TODO Authors—Can you explain this (1,x,y)/(0,x,y) syntax more? #}
+
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - thoughts on this?

Reply:
MDN explains specificity in terms of "hundreds, tens and ones" so if x was 3 and y was 4 in that example then (1,x,y) would be 1 hundreds, 3 tens, 4 ones - the number 134. But this leads authors to think that, for example, 12 class selectors (x=12) is more specific than one ID selector - 12 *10 = 120 > 100. So CSS experts tend to write the specificity with the number of ID, class, and element/pseudo-element selectors as a list: (ID, class, element).

Reply:
Is there somewhere we can link to this for other readers who are similarly confused? Don't see any mention of this notation in https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
The spec? :) https://www.w3.org/TR/selectors/#specificity-rules

Reply:
Like it. Added a sentence to this in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1846/commits/d83e68afcf451c0735bddf8b527fb0b578be9435. Marking as resolved but feel free to reopen this and suggest alternative language.